infinispan-embedded/src/main/java/org/hibernate/ogm/datastore/infinispan/InfinispanDialect.java

```diff
@@ -228,6 +247,104 @@ public void forEachTuple( ModelConsumer consumer, TupleTypeContext tupleTypeCont
 		return (KeyProvider<EK, AK, ISK>) provider.getKeyProvider();
 	}
 
+	@Override
+	public ClosableIterator<Tuple> callStoredProcedure( String storedProcedureName, ProcedureQueryParameters queryParameters, TupleContext tupleContext ) {
+		validate( queryParameters );
+		EmbeddedCacheManager embeddedCacheManager = getCacheManager().getCacheManager();
+		Cache<String, String> cache = embeddedCacheManager.getCache( "___stored_procedures", true );
```

Comment:
Maybe we can extract "___stored_procedures" constants, used in API to register the alias procedure name.